website/package.json

```diff
@@ -15,17 +15,17 @@
     "archive": "node archive.js"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.21",
-    "@docusaurus/preset-classic": "2.0.0-beta.21",
-    "@easyops-cn/docusaurus-search-local": "^0.28.0",
+    "@docusaurus/core": "^2.0.1",
+    "@docusaurus/preset-classic": "^2.0.1",
+    "@easyops-cn/docusaurus-search-local": "^0.26.1",
```

Comment:
did you verify that the search still works? Let's pin down the dependencies as well?

Reply:
Yes, it does.